things/postgres/groups.go

```diff
@@ -523,11 +463,90 @@ func (gr groupRepository) UnassignMember(ctx context.Context, groupID string, id
 	return nil
 }
 
-type dbMember struct {
-	MemberID  string    `db:"member_id"`
-	GroupID   string    `db:"group_id"`
-	CreatedAt time.Time `db:"created_at"`
-	UpdatedAt time.Time `db:"updated_at"`
+func (gr groupRepository) retrieve(ctx context.Context, ownerID string, pm things.PageMetadata) (things.GroupPage, error) {
+	var ownq string
+	if ownerID != "" {
+		ownq = "owner_id = :owner_id"
```

Comment:
Can we use `getOwnerQuery` here?

Reply:
I don't think so.  In group SQL table we have `owner_id` and in channel SQL tale `owner`

Reply:
Right!